ntbs-service/Helpers/UrlHelper.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using Castle.Core.Internal;
+
+namespace ntbs_service.Helpers
+{
+    public static class UrlHelper
+    {
+        public static string SanitiseHttpsUrl(string url)
+        {
+            if (url.IsNullOrEmpty())
```

Comment:
I assume this is here because `Uri.TryCreate` throws for a null url?

Reply:
I assumed it does, but it doesn't actually. I've removed it, thanks for prompting me